client/dashboard/sites/settings-site-visibility/index.tsx

```diff
@@ -3,9 +3,9 @@ import { Card, CardBody } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { useState } from 'react';
 import { siteQuery, siteSettingsMutation, siteSettingsQuery } from '../../app/queries';
-import { Notice } from '../../components/notice';
 import PageLayout from '../../components/page-layout';
 import SettingsPageHeader from '../settings-page-header';
+import SitePreviewLinks from '../site-preview-links';
```

Comment:
Can we lazy load it? It introduces the `moment` dependency and I don't think it's required when the preview is disabled.

Reply:
The `settings/site-visibility` route is already lazy-loaded, so I think it's enough (?) and this setting will be very rarely visited by the user.

In any case, moment will gone after DOTDASH-2 is fixed.